packages/rum-core/src/domain/view/viewMetrics/trackInteractionToNextPaint.ts

```diff
@@ -76,6 +84,20 @@ export function trackInteractionToNextPaint(
       } else {
         interactionToNextPaintTargetSelector = undefined
       }
+      if (
+        !interactionToNextPaintTargetSelector &&
+        isExperimentalFeatureEnabled(ExperimentalFeature.NULL_INP_TELEMETRY)
+      ) {
+        if (interactionSelectorMap.has(newInteraction.startTime)) {
+          interactionToNextPaintTargetSelector = interactionSelectorMap.get(newInteraction.startTime)
+          interactionSelectorMap.delete(newInteraction.startTime)
+        }
+      }
+      interactionSelectorMap.forEach((_, key) => {
+        if (key > 10 * ONE_SECOND) {
```

Comment:
**🔨 warning:** ‏The key being a relative time, with this condition we'll keep only entries that occur within the 10 first seconds of the page. What you are looking for is:

```suggestion
        if (relativeNow() - key > 10 * ONE_SECOND) {
```

Reply:
Good Catch! 😭